src/persistence/settings.cpp

```diff
@@ -226,9 +224,7 @@ void Settings::loadGlobal()
     s.endGroup();
 
     s.beginGroup("Chat");
-    {
         chatMessageFont = s.value("chatMessageFont", Style::getFont(Style::Big)).value<QFont>();
-    }
```

Comment:
Please leave those brackets. You can add those rather after/before any beginGroup/endGroup. It serves two purposes:

1. Marks code as a block.
2. Indents correctly with auto-indent.

So, please leave it.